### ISSUE TITLE
no appium logs in the console

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/appium#293]
it seems happend after adding `-use-plugin images`

**Comment by vdelendik** [comment]
root cause is inalid data detection using `ios info` and jq when timeoud out error displayed:
```+ deviceClass='null
null
null
null
null
null
null
iPhone'
```


the same for the version:
```
+ PLATFORM_VERSION='null
null
null
null
null
null
null
15.7.1'
```

as result invalid node configuration is provided to the appium and it crashes asap.
as a fix we have to ignore null values to skip all json messages where deviceClass or Version not available (error messages)

**Comment by vdelendik** [comment]
fixed in 2.0.9-beta7

**Comment by dhreben** [comment]
Fixed
